stdlib/event/event.lua

```diff
@@ -252,6 +220,38 @@ function Event.remove(event_id, handler, matcher, pattern)
     return Event
 end
 
+-- A dispatch helper function
+--
+-- Call any matcher and, as applicable, the event handler, in protected mode.  Errors are
+-- caught and logged to stdout but event processing proceeds thereafter; errors are suppressed.
```

Comment:
Should read
Errors are surpressd and events will continue to process as long as someone is online